### PR TITLE
Added js code for tag manager and field for tag manager code in admin

### PR DESCRIPTION
**Pull request by edwoodward**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@
       OX.url_prefix = "<%= request.script_name %>";
     </script>
 
+    <%= render partial: "layouts/google_tag_manager" %>
```
Comment (BryanEliDimas) [inline app/views/layouts/application.html.erb:11]: Did you mean to add this the new flow? https://github.com/openstax/accounts/blob/master/app/views/layouts/newflow_layout.html.erb

Only adding it to app/views/layouts/application.html.erb will make it only available in the legacy flow and users' profile.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -33,7 +35,7 @@
   </head>
 
   <body class="<%= controller_name %> <%= action_name %>">
-
+    <%= render partial: "layouts/tag_manager_snippet" %>
```
Comment (BryanEliDimas) [inline app/views/layouts/application.html.erb:38]: Same as above: Did you mean to add this the new flow? https://github.com/openstax/accounts/blob/master/app/views/layouts/newflow_layout.html.erb

Only adding it to app/views/layouts/application.html.erb will make it only available in the legacy flow and users' profile.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (edwoodward) [reply app/views/layouts/application.html.erb:11]: Good point. Fixed

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (edwoodward) [reply app/views/layouts/application.html.erb:38]: Fixed

**Pull request merged**